mule-migration-tool-library/src/test/resources/mule/apps/core/processor-chain-02-original.xml

```diff
@@ -5,9 +5,15 @@
       xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
 
     <flow name="testFlow">
-        <processor-chain name="Processor Chain">
+        <processor-chain>
```

Comment:
can we add a test that has multiple processor chains without a name?

Reply:
there are two unnamed processor chains in the test

Reply:
sorry I got confused and thought the second one was the closing tag. Ignore this comment